addons/bus/models/res_users.py

```diff
@@ -17,3 +17,6 @@ def _compute_im_status(self):
         }
         for user in self:
             user.im_status = presence_by_user.get(user, "offline")
+
+    def _get_online_status(self):
```

Comment:
This would require a little bit of explaining.

Because `leave_online` is technically "online" in regards to `bus`, but we don't want to include it here.

I would suggest to use a different name, maybe `available`? And it should only be `True` or `False` probably.

Reply:
@seb-odoo Is this better ?